jekyll/_includes/snippets/runner/machine-runner-deb-package-install.adoc

```diff
@@ -0,0 +1,36 @@
+//[#machine-runner-debian-package-installation]
+//== Machine runner debian package installation
+
+The easiest way to install runner on a debian or ubuntu system is to use CircleCI's pre-built packages.
+
+
+. On the target machine, install the CircleCI registry. This script will automatically run `apt-get update`.
++
+```shell
+curl -s https://packagecloud.io/install/repositories/circleci/runner/script.deb.sh?any=true | sudo bash
+
+```
+
+. Next, install the `circleci-runner` package.
++
+```shell
+sudo apt-get install -y circleci-runner
+
+```
+
+. Open the runner config YAML file and replace `<< AUTH_TOKEN >>` with your token, using the text editor of your choice.
++
+```shell
+sudo vim /etc/circleci-runner/circleci-runner-config.yaml
```

Comment:
Can we use a more universal editor than `vim`? Perhaps just use `sed`?

Reply:
We should move to `vi` if we want it to be universal (system with vim baked in will likely alias vi to vim any way), but otherwise its a bit more common to use `nano` since it's pretty ubiquitous as well but less arcane for the uninitiated. 

Big fan of just using sed to avoid all that though. Something along the lines of

```
sed -i 's|<< AUTH_TOKEN >>|YOUR_TOKEN_HERE|g' /etc/circleci-runner/circleci-runner-config.yaml
```
should do the trick

Reply:
+1 to a one-liner with `sed`.